Lib/pyclbr.py

```diff
@@ -52,40 +53,41 @@
 
 class _Object:
     "Information about Python class or function."
-    def __init__(self, module, name, file, lineno, parent):
+    def __init__(self, module, name, file, lineno, end_lineno, parent):
```

Comment:
Since `_Object` is private, its signature does not have to be changed.  Neither `end_lineno` nor `parent` are optional here, as they get the defaults from the subclasses.  Arguments for `super_` and `is_async` are not passed on to `_Object`.  So its signature cannot match that of the subclasses.